R/which_element.R

```diff
@@ -0,0 +1,42 @@
+#' which_element
+#'
+#' This function returns locations within an EML list object that contain an instance where \code{element == value}
```

Comment:
Another nitpick: what do you think about changing this to "returns indices within an EML list...", basically just substitute locations for indices.   
I was unclear on what this function did until I ran it.